application/src/user.h

```diff
@@ -0,0 +1,20 @@
+#ifndef USER_H
+#define USER_H
+
+#include <QString>
+
+struct User {
+    User() = default;
+    User(QString userId, QString userName) : id(userId), name(userName) {}
+
+    bool isValid() const
```

Comment:
```suggestion
    bool isValid() const {
```
сохраняй единый стиль